docs/UserGuide.md

```diff
@@ -441,24 +431,47 @@ Show a message explaining how to access the help page.
 
 Format: `help`
 
-#### 5.4.2 Exit MyInsuRec: `exit`
+#### 5.4.2 Exit *MyInsuRec*: `exit`
 
 Exit the program.
 
 Format: `exit`
 
+#### 5.4.3 Clear *MyInsuRec*: `clear`
+
+Format: `clear`
+
+This is a one-shot way of removing all data stored in *MyInsuRec*.
+Only clear when you are sure you do not need the data anymore, or else make a [backup](#553-creating-and-using-data-file-backups).
+
+Example usage: 
+* Clearing default data shipped with *MyInsuRec*
+* Restarting the app from an empty version
+
 ### 5.5 Others
 
 #### 5.5.1 Save the data
 
-MyInsuRec data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+*MyInsuRec* data is saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
 
 #### 5.5.2 Edit the data file
 
-MyInsuRec data are saved as a JSON file `[JAR file location]/data/myinsurec.json`. Advanced users are welcome to update data directly by editing that data file.
+*MyInsuRec* data are saved as a JSON file `[JAR file location]/data/myinsurec.json`. Advanced users are welcome to update data directly by editing that data file.
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-If your changes to the data file makes its format invalid, MyInsuRec will discard all data and start with an empty data file at the next run.
+If your changes to the data file makes its format invalid, *MyInsuRec* will discard all data and start with an empty data file at the next run.
+</div>
+
+#### 5.5.3 Creating and using data file backups
+
+This is the recommended method of storing backups of the `[JAR file location]/data/myinsurec.json` file.
+First, create a copy of the original file in the `[JAR file location]/data` folder and rename it to `myinsurec_date.json` where date identifies when the file was created.
```

Comment:
for the `date` here are you refering to `myinsurec_29102022`?

Reply:
yes, its really up to the user how they want to name their files but I thought giving suggestion might help. I'll add the example you have to the guide.

Reply:
would prefer a format like `myinsurec_DATE.json` where `DATE` is the date where the file is created or sth like that